pkgs/tools/misc/Archi/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchurl }:
+
+let version = "4.7.1";
+in stdenv.mkDerivation {
+
+  pname = "Archi";
+  inherit version;
+
+  src =
+    if stdenv.hostPlatform.system == "x86_64-linux" then
+      fetchurl {
+        url = "https://www.archimatetool.com/downloads/archi/Archi-Linux64-${version}.tgz";
+        sha256 = "0sd57cfnh5q2p17sd86c8wgmqyipg29rz6iaa5brq8mwn8ps2fdw";
+      }
+    else if stdenv.hostPlatform.system == "x86_64-linux" then
+      throw "Unsupported system"
+      /*
+       * A Mac version is available; needs to be tested on a Mac system:
+       * fetchurl {
+       *   url = "https://www.archimatetool.com/downloads/archi/Archi-Mac-${version}.zip";
+       *   sha256 = "0zg0bs9i2kkr11bbssfxhrp4ym2zzhb2bk7m38z9cj99h7pmgyhr";
+       * }
+       */
+    else
+       throw "Unsupported system";
+
+  installPhase = ''
+    cp -r . $out
+    mkdir -p $out/bin
+    ln -s $out/Archi $out/bin/Archi
```

Comment:
This is creating an invalid relative symlink to `../Archi` under NixOS, but it works correctly on Nix+Linux.